65black/StackOverflow-lite
# #160167237 Fix NaN userid in Database When POST Questions With Invalid Token

#### What does this PR do?
Currently, when a user tries to post a question with an invalid token, the verifyToken.js middleware correctly throws a 401 status code but the question posts anyway with NaN as the userId.
This PR aims to fix that.

#### Description of Task to be completed
Fixes a bug on the following endpoint:

```
POST api/v1/questions/
```
#### How should this be manually tested?
After cloning the repo and installing all dependencies with `npm install`

- Using Postman, make a post request to the above endpoint with an invalid token
- It should fail with a 401 error
- Now check your database and confirm that your post request was queried to the database.

#### What are the relevant pivotal tracker stories?
#160167237
